src/mem/AddrMapper.py

```diff
@@ -37,19 +37,19 @@
 from m5.SimObject import SimObject
 
 # An address mapper changes the packet addresses in going from the
```

Comment:
We should move this comment to be a true python documentation comment:

E.g.,

```
"""An address mapper changes...
"""
```

And it should be under `class AddrMapper`

Finally, it would be good to mention here that you can use the `splice` function to insert this address mapper between two already connected objects.